addons/mail/static/tests/chat_window/chat_window.test.js

```diff
@@ -1076,3 +1076,20 @@ test("open channel in chat window from push notification", async () => {
     );
     await contains(".o-mail-ChatWindow", { text: "General" });
 });
+
+test("Ctrl + K opens command palette in conversation mode in chat window", async () => {
```

Comment:
I would rename to `Ctrl+k opens the @ command palette`.

Current name is fine, but if there's an equivalent naming that is shorter it improves readability.